### PR TITLE
Merge config files

### PR DESCRIPTION
**Pull request by alvin1221**

### CODE REVIEW

**Comment by JaylinYu** [comment]
wait until the bridging online/offline msg is merged

**Review by JaylinYu — APPROVED** [review]

**Comment by alvin1221** [comment]
> 

Done

**Review by JaylinYu — APPROVED** [review]

**Comment by JaylinYu** [comment]
goooooooooooooooood

**Pull request merged**
